linode_api4/objects/base.py

```diff
@@ -144,20 +145,36 @@ def __setattr__(self, name, value):
         """
         Enforces allowing editing of only Properties defined as mutable
         """
-        if name in type(self).properties.keys() and not type(self).properties[name].mutable:
-            raise AttributeError("'{}' is not a mutable field of '{}'"
-                .format(name, type(self).__name__))
+        if name in type(self).properties.keys():
+            if not type(self).properties[name].mutable:
+                raise AttributeError("'{}' is not a mutable field of '{}'"
+                    .format(name, type(self).__name__))
+
+            self._changed = True
+
         self._set(name, value)
 
-    def save(self):
+    def save(self, force=True) -> bool:
```

Comment:
This is by default `True` to maintain backwards compatibility, just incase people are manually populating a resource and saving it.